config/betterquesting/DefaultQuests.json

```diff
@@ -94586,24 +94586,14 @@
     "916:10": {
       "questID:3": 916,
       "preRequisites:11": [
-        537,
-        563,
-        909,
-        910,
-        911,
-        912,
-        913,
-        914,
-        915,
-        985,
-        2077
+        563
       ],
       "properties:10": {
         "betterquesting:10": {
           "snd_complete:8": "random.levelup",
           "taskLogic:8": "AND",
           "partySingleReward:1": 0,
-          "visibility:8": "SECRET",
+          "visibility:8": "NORMAL",
```

Comment:
@chochem why you set to visible ?

Reply:
killing the wither shouldnt exactly be a secret task. In fact its part of the main progression. (killing twice is a different quest and still secret)